README.md

```diff
@@ -1,8 +1,17 @@
 # ClusterManagers
 
-Support for different job queue systems commonly used on compute clusters.
+> [!WARNING]
```

Comment:
Might be a good idea to grab a repostatus.org badge. "Unsupported" or "Suspended" seem like likely fits?

Reply:
Wonder if we could have it at a finer level, like "Unsupported" on the different individual cluster managers? Since it seems like from the discourse thread that there are certain managers that work well.

Reply:
Fwiw I use and try to help maintaining the lsf manager.

Reply:
Indeed it sounds like lsf is known to work well. Any others? We could just have a column with whether a method works or not

Reply:
Condor works with some common filesystem mounted assumptions

Reply:
>Wonder if we could have it at a finer level, like "Unsupported" on the different individual cluster managers

I'm in principal in favor of this, the trouble is that it requires active, ongoing assessment. Having really granular badges for working or not working implies a level of insight that I just don't think we can have without tests / CI.

Which brings us back to #105 :wink:

Reply:
Could it be worth it to also have a clause explaining that most cluster managers are not expected to be guaranteed to work out of the box. I need to supply quite a few extra program flags to the LSF manger and I also need a separate hack to find an open port for the workers for it to work on my system. I don't consider it a bug or deficit that ClusterManagers can't find a set of working arguments for me automatically. 

For the maintenance status question:
Would it be useful if the level of support is indicated through a column with maintainer names (with something like "maintainer needed" indicates that it is not well maintained)?

Reply:
@DrChainsaw I think both of those ideas are really good.

Reply:
Where does this stand - did you want to add that stuff to the table in the readme?

Reply:
Sounds good to me. Sorry I didn’t adjust the pr yet; just busy with teaching